webapp/src/components/HomePage/HomePage.js

```diff
@@ -19,6 +22,11 @@ export default class HomePage extends React.PureComponent {
     this.props.onFetchPublications()
   }
 
+  onLearnMore = () => {
+    const { onNavigate } = this.props
+    onNavigate(locations.auction())
```

Comment:
I know `onNavigate` was already on the container but I think it was legacy (it wasn't being used). I think it's a bit better to delegate the `locations.auction` to the container and delete it:

```suggestion
    this.props.onNavigateToAuction()
```

**BUT**, maybe the `<Button primary={true} onClick={this.onLearnMore}>` can just be a link?

```
<Link to={locations.auction()}>
  <Button primary={true}>
```

So in conclusion, using a link and removing the method from the container